climpred/graphics.py

```diff
@@ -1,11 +1,13 @@
 import warnings
 from collections import OrderedDict
+from typing import Optional, Tuple, Union
 
 import numpy as np
 import xarray as xr
 from xarray.coding.times import infer_calendar_name
 
 from .checks import DimensionError
+# from .classes import HindcastEnsemble, PerfectModelEnsemble
```

Comment:
rm

create alias if imported else Any

Reply:
from typing import TYPE_CHECKING
from queue import Queue

if TYPE_CHECKING:
    BaseQueue = Queue[str]  # this is only processed by mypy
else:
    BaseQueue = Queue  # this is not seen by mypy but will be executed at runtime